### ISSUE TITLE
xlights Crashes when exporting 6x6 P5 Matrix to LOR Clipboard

### ISSUE THREAD

**Issue by PierceLights** [smeighan/xLights#3552]
**Describe the bug**
I am trying to export a Matrix (6x6 P5 Panel) to LOR Clipboard.  Every time that I try it crashes with a DUMP (attached).

**To Reproduce**
1. Open Sequence
2. Click on Render All in Toolbar
3. Choose the P5 Panel, Right Mouse and choose Model > Render and Export or Model > Export ... both options FAIL.

**Expected behavior**
Expect LOR Clipboard file to be produced.

**Versions (please complete the following information):**
 - OS: Windows 10
 - xLights version: 2022.22 (also received error on 2022.21)

**Attachments**
I have attached three zip files produced by xLights the older two from xLights 2022.21 and later one from xLights 2022.22.

[xLights_dbgrpt-19452-20221111T205258.zip](https://github.com/smeighan/xLights/files/9993881/xLights_dbgrpt-19452-20221111T205258.zip)
[xLights
[xLights_dbgrpt-57428-20221105T163701.zip](https://github.com/smeighan/xLights/files/9993883/xLights_dbgrpt-57428-20221105T163701.zip)
_dbgrpt-46960-20221111T205604.zip](https://github.com/smeighan/xLights/files/9993882/xLights_dbgrpt-46960-20221111T205604.zip)

**Comment by PierceLights** [comment]
Previously I had only tried exporting the P5 Matrix.  I have now tried exporting other props/models and also from other sequences form which I had exported before.  They all result in a crash.

**Comment by PierceLights** [comment]
Status Update ... I did the following:
1.  Had my wife see if she coudl export to LOR Clipboard file?  She can.  She is on 2022.13 Version
2. Obtained my wife's rgbeffects file and opened an old sequence and tried to export a model to LOR Clipboard.  Same crash as before.
3.  I uninstalled xLights and installed the 2022.13 version.  Using this version and my old working directory with NO changes, I can now export to LOR Clipboard all models (including P5).
So it seems the issue must have happened somewhere between 2022.13 and 2022.21.
